api/src/main/java/org/apache/brooklyn/api/objs/Configurable.java

```diff
@@ -41,6 +41,11 @@
     @Deprecated
     public <T> T setConfig(ConfigKey<T> key, T val);
 
+    /**
+     * Convenience method for {@code config().get(key)}
+     */
+    <T> T getConfig(ConfigKey<T> key);
```

Comment:
I moved it because it was confusing the compiler when in multiple interfaces, even though the signature was the same it complained of a local generics `<T>` mismatch.  Have removed from a couple places.